src/ViewHelpers/CarsViewHelper.php

```diff
@@ -0,0 +1,24 @@
+<?php
+require_once 'src/Models/CarsModel.php';
+class CarsViewHelper
+{
+    public static function displayAllCars(array $carObjs)
+    {
+        if (count($carObjs) < 1)
```

Comment:
Picky thing, make this === 0 instead

For some reason saying less than 1 feels strange